src/PBE/ExecutableContainer.cs

```diff
@@ -112,9 +117,10 @@ private bool TryGetFSVersionPath(string basePath, FSVersion fsVersion, out strin
 
             // Beta Version auslesen
             int maxBeta = -1;
-            foreach (string path in fsDirs.Where(d => Regex.IsMatch(Path.GetFileName(d), @"(.?Beta(?<beta>\d*))$", RegexOptions.IgnoreCase)))
+            string betaRegexPattern = @"(.?Beta\s*(?<beta>\d*))$";
+            foreach (string path in fsDirs.Where(d => Regex.IsMatch(Path.GetFileName(d), betaRegexPattern, RegexOptions.IgnoreCase)))
             {
-                Match m = Regex.Match(Path.GetFileName(path), @"(.?Beta(?<beta>\d*))$");
+                Match m = Regex.Match(Path.GetFileName(path), betaRegexPattern, RegexOptions.IgnoreCase);
```

Comment:
Nur eine kleine Anmerkung - du brauchst das nicht ändern.
In solchen Fällen ist es ggf. sinnvoll `RegexOptions.IgnoreCase` direkt in den Regex zu integrieren, denn diese Option hängt unmittelbar mit dem Pattern zusammen. Dafür gibt es 2 Optionen:

1.  Den Regex als Instanz speichern, die wiederverwendet wird. Hier allerdings weniger Optimal, denn wenn es keine passenden Ordner gibt braucht man den Regex nicht und dann hätten wir ihn umsonst instanziiert.
2. Die Options direkt in das Pattern packen: `(?i)(.?Beta\s*(?<beta>\d*))$`
  siehe auch <https://learn.microsoft.com/dotnet/standard/base-types/regular-expression-options>